### PR TITLE
Fix csi migration feature gates

### PR DESCRIPTION
**Pull request by Jiawei0227**
Update CSI Migration related feature gates to the latest status.

/cc @msau42  @sftim
/sig storage

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -211,6 +211,18 @@ different Kubernetes components.
...
+| `CSIMigrationAzureDiskComplete` | - | Alpha | 1.17 | 1.20 |
+| `CSIMigrationAzureDiskComplete` | - | Deprecated | 1.21 | - |
+| `CSIMigrationAzureFileComplete` | - | Alpha | 1.17 | 1.20 |
+| `CSIMigrationAzureFileComplete` | - | Deprecated |  1.21 | - |
+| `CSIMigrationGCEComplete` | - | Alpha | 1.17 | 1.20 |
+| `CSIMigrationGCEComplete` | - | Deprecated | 1.21 | - |
+| `CSIMigrationOpenStackComplete` | - | Alpha | 1.17 | 1.20 |
+| `CSIMigrationOpenStackComplete` | - | Deprecated | 1.21 | - |
+| `CSIMigrationvSphereComplete` | - | Beta | 1.19 | 1.21 |
+| `CSIMigrationvSphereComplete` | - | Deprecated | 1.22 | - |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:225]: Should we add some text to the descriptions of these feature gates to (later in the same page) to explain what happened to them?

**Comment by reylejano** [comment]
@Jiawei0227 , please rebase this PR. Also PTAL at [@sftim 's comment above](https://github.com/kubernetes/website/pull/28629#discussion_r667404937)

**Comment by PI-Victor** [comment]
@Jiawei0227 please rebase this PR, the PR docs deadline is tomorrow, thanks!

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (Jiawei0227) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Done.

**Comment by Jiawei0227** [comment]
@reylejano @PI-Victor @sftim 

Sorry for being so late on this. I have been pretty busy these days. Thanks a lot for your review on this!

**Pull request merged**